frontend/pages/register/index.tsx

```diff
@@ -1,3 +1,12 @@
+import Link from "next/link"
+
 export default function Register() {
-  return <div className="Register">I AM REGISTRATION!!!!!!</div>
+  return (
+    <>
+      <div className="Register">I AM REGISTRATION!!!!!!</div>
+      <Link href="/register/passport">
+        <a>Register for a Passport account</a>
+      </Link>
```

Comment:
Is this also outdated perhaps? I see in the thread above that this was meant to be a placeholder && it should've been replaced with `pages/viewer-registration/viewer-registration.tsx` with the last merge

Reply:
Also appears to be so! Apologies, the "file changes" tab doesn't feel super intuitive for review